.github/workflows/refapp-2x-open-Concept-Lab.yml

```diff
@@ -0,0 +1,40 @@
+name: RefApp 2.x Open Concept Lab
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+  repository_dispatch:
+    types: [qa]
+  workflow_dispatch:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    services:
+      firefox:
+        image: selenium/node-firefox:3.141.59-gold
+        env:
+          HUB_HOST: hub
+          HUB_PORT: 4444
+    steps:
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Checkout qaframework
+        uses: actions/checkout@master
+        with:
+          repository: ${{github.repository}}
+      - name: run db and web containers
+        run: |
+          docker-compose -f docker/docker-compose-refqa.yml up -d
+      - name: wait for openmrs instance to start
+        run: while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' http://localhost:8080/openmrs/login.htm)" != "200" ]]; do sleep 1; done
+      - name: Run qaframework on selenium
+        run: |
+          echo "cucumber.publish.enabled=true" > src/test/resources/cucumber.properties
```

Comment:
Great work @jwnasambu look at https://github.com/openmrs/openmrs-contrib-qaframework/blob/c7b2070c5e22e1c6d9693863980f3c24f5d74083/.github/workflows/qa.yml#L35 and https://github.com/openmrs/openmrs-contrib-qaframework/blob/c7b2070c5e22e1c6d9693863980f3c24f5d74083/.github/workflows/qa.yml#L37 for reference or better yet update your branch with master changes and adjust the .yml file abit.

Reply:
@mherman22 Thanks so so much! Let me do it right away!